packages/react-composites/src/composites/MeetingComposite/MeetingComposite.tsx

```diff
@@ -102,20 +104,39 @@ const MeetingScreen = (props: MeetingScreenProps): JSX.Element => {
 
   const [currentMeetingState, setCurrentMeetingState] = useState<CallState>();
   const [currentPage, setCurrentPage] = useState<MeetingCompositePage>();
+  const [unreadChatMessagesCount, setUnreadChatMessagesCount] = useState<number>(0);
+  const [showChat, setShowChat] = useState(false);
+  const [showPeople, setShowPeople] = useState(false);
+
   meetingAdapter.onStateChange((newState) => {
     setCurrentPage(newState.page);
     setCurrentMeetingState(newState.meeting?.state);
   });
 
-  const [showChat, setShowChat] = useState(false);
-  const [showPeople, setShowPeople] = useState(false);
+  useEffect(() => {
+    const incrementUnreadtChatMessagesCount = (event: { message: ChatMessage }): void => {
+      if (!showChat && event.message.senderDisplayName !== '') {
```

Comment:
Please extract a local helper function that better states what this check does:

```ts
const isMessageFromBot = (message): bool => {
  // Some comment about why display name is a good heuristic. Is it?
  return message.senderDisplayName === '';
}
```

Reply:
Added a comment about this, but in short system messages wont have a displayname so we dont have to have those notifications added.